src/main/scala/group/holmes/analytics/actors/AnalyticsSupervisor.scala

```diff
@@ -0,0 +1,28 @@
+package group.holmes.analytics.actors
```

Comment:
we bought some DNS names recently. holmes.group, holmes-processing.com etc. 

This comment is so we select one and then set a standard.